vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomFragment.kt

```diff
@@ -51,11 +52,13 @@ import javax.inject.Inject
 @Parcelize
 data class CreateRoomArgs(
         val initialName: String,
-        val parentSpaceId: String? = null
+        val parentSpaceId: String? = null,
+        val isSubSpace: Boolean = false
 ) : Parcelable
 
 class CreateRoomFragment @Inject constructor(
         private val createRoomController: CreateRoomController,
+        private val createSpaceController: CreateSubSpaceController,
```

Comment:
Why having 2 controllers? generally 1 recycler  = 1 controller

Reply:
They are quite different, I think it's clearer to have two instead of several if/else in controller